client/signup/steps/user/index.jsx

```diff
@@ -404,6 +406,32 @@ export class UserStep extends Component {
 		return headerText;
 	}
 
+	getRedirectToAfterLoginUrl() {
```

Comment:
There seems to be another function in this file with the same name `getRedirectToAfterLoginUrl(). Could we please roll over the changes from this new function to the existing function?